services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/mapper/annotations/DynamoDbAttribute.java

```diff
@@ -40,11 +39,10 @@
  *      }
  *
  *      public void setInternalKey(String internalKey) {
- *          return this.internalKey = internalKey;}
+ *          return this.internalKey = internalKey;
```

Comment:
Is this example code correct? This is a void method (and a setter), so probably shouldn't be returning here.

Reply:
I noticed the extra curly, but not the `return` on a setter 🤦🏻‍♀️
Fixed now.